core/src/main/java/org/akaza/openclinica/dao/extract/OdmExtractDAO.java

```diff
@@ -2291,9 +2291,9 @@ public void getClinicalData(StudyBean study, DatasetBean dataset, OdmClinicalDat
 
         this.setEventGroupItemDataWithUnitTypesExpected();
         logger.debug("Begin to GetEventGroupItemWithUnitSql");
-        ArrayList viewRows = select(getEventGroupItemWithUnitSql(studyIds, sedIds, itemIds, dateConstraint, datasetItemStatusId, studySubjectIds));
+        ArrayList viewRows = select(getEventGroupItemWithUnitSqlRewritten(studyIds, sedIds, itemIds, dateConstraint, datasetItemStatusId, studySubjectIds));
```

Comment:
Is getEventGroupItemWithUnitSql method still being used elsewhere? If not, can we delete it completely?

Reply:
all done, thanks